spec/controllers/v0/pension_claims_controller_spec.rb

```diff
@@ -21,7 +21,6 @@ def logger_mock(str, x)
       expect(Rails.logger).to receive(:info).with('21P-527EZ submission to Sidekiq success',
                                                   hash_including(:confirmation_number, :user_uuid,
                                                                  :in_progress_form_id))
-      expect(Rails.logger).to receive(:info).at_least(:once)
```

Comment:
This test should verify a specific log entry. The current implementation passes with any `info` log, regardless of relevance. Previously, it was passing due to an unrelated `ActiveModel::Serializer` render log. The relevant logs are tested earlier in the spec.